packages/api/lib/rulesHelpers.js

```diff
@@ -55,12 +55,10 @@ async function queueMessageForRule(rule, eventObject, eventSource) {
     ? collectionInNotification
     : rule.collection;
   const item = {
-    workflow: rule.workflow,
-    provider: rule.provider,
+    ...rule,
```

Comment:
I could have just added `rule.queueUrl` to the object passed into `Rule.buildPayload`, but I saw no reason not to pass **all rule properties except those which are overwritten**, since `Rule.buildPayload` expects a rule object anyway